core/src/main/java/org/sonarsource/sonarlint/core/analyzer/issue/IssueFilters.java

```diff
@@ -28,21 +29,39 @@
 @BatchSide
 public class IssueFilters {
   private final IssueFilter[] filters;
+  private final org.sonar.api.issue.batch.IssueFilter[] deprecatedFilters;
   private final Project project;
 
-  public IssueFilters(Project project, IssueFilter[] exclusionFilters) {
+  public IssueFilters(Project project, IssueFilter[] exclusionFilters, org.sonar.api.issue.batch.IssueFilter[] filters) {
     this.project = project;
     this.filters = exclusionFilters;
+    this.deprecatedFilters = filters;
+  }
+
+  public IssueFilters(Project project, IssueFilter[] filters) {
+    this(project, filters, new org.sonar.api.issue.batch.IssueFilter[0]);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this use of "IssueFilter"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)